lms/templates/courseware/gradebook.html

```diff
@@ -74,13 +74,21 @@
         <thead>
           <tr> <!-- Header Row -->
             %for section in templateSummary['section_breakdown']:
-              <th><div class="assignment-label">${section['label']}</div></th>
+              <%
+                tooltip_str = section['detail']
+                if "=" in section['detail']:
+                    tooltip_str = section['detail'][0: section['detail'].rfind('=')]
+
+                if "-" in tooltip_str:
+                    tooltip_str = tooltip_str[0: tooltip_str.rfind('-')]
+              %>
+              <th title="${tooltip_str}"><div class="assignment-label">${section['label']}</div></th>
             %endfor
-            <th><div class="assignment-label">Total</div></th>
+            <th title="${_(' Total ')}"><div class="assignment-label">Total</div></th>
```

Comment:
Why are you adding extra spaces here? If there's a good reason, you should add the extra spaces outside of the translation string; translators will certainly accidentally strip off the extra spaces.

However, if you're doing it for style, this si a bad idea. It may look bad in a long language like German. You should do stylistic changes via CSS.